From eec70c47f89fd0187a555964962c9a42b2965103 Mon Sep 17 00:00:00 2001
From: Michael Sarahan <msarahan@gmail.com>
Date: Tue, 12 Jun 2018 17:20:24 -0500
Subject: [PATCH] only build UMFPACK library

---
 UMFPACK/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/UMFPACK/Makefile b/UMFPACK/Makefile
index b87a5bb..6b24cbb 100644
--- a/UMFPACK/Makefile
+++ b/UMFPACK/Makefile
@@ -5,7 +5,7 @@
 SUITESPARSE ?= $(realpath $(CURDIR)/..)
 export SUITESPARSE
 
-default: all
+default: library
 
 include ../SuiteSparse_config/SuiteSparse_config.mk
 
-- 
2.11.1

